source/tyr/tyr/binarisation.py

```diff
@@ -1125,9 +1125,47 @@ def poi2mimir(self, instance_name, input, autocomplete_version, job_id=None, dat
 @celery.task(bind=True)
 def fusio2s3(self, instance_config, filename, job_id, dataset_uid):
     """Zip fusio file and launch fusio2s3"""
+    filename = enrich_ntfs_with_addresses(filename, job_id, dataset_uid)
     _inner_2s3(self, "fusio", instance_config, filename, job_id, dataset_uid)
 
 
+def enrich_ntfs_with_addresses(filename, job_id, dataset_uid):
+    """launch enrich-ntfs-with-addresses"""
+
+    job = models.Job.query.get(job_id)
+    dataset = _retrieve_dataset_and_set_state("fusio", job.id)
+    instance = job.instance
+
+    logger = get_instance_logger(instance, task_id=job_id)
+    filename = zip_if_needed(filename)
+
+    try:
+        params = [
+            "--input",
+            filename,
+            "--output",
+            filename,
```

Comment:
I am not sure if this is a good idea to replace the input ntfs with the enriched one.
Maybe write to a different location ?

Reply:
it works and it should not be a problem imho.
if there is a problem, filename should not be atered by the output.
if the job succeed, the ntfs is just replaced.

Reply:
ok I see, the ntfs is sent to fusio2ed after, we should keep the original; ok

Reply:
Yeah I don't know exactly what happen with the ntfs (not when), but I know that *some* things can happen (sent to fusio2ed, moved to a backup directory). 
So better be safe than sorry :)

Reply:
fixed in the last commit.
How can I increase the code coverage ? Sonar is warning